specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/examples/UpdateBillingAccount.json

```diff
@@ -0,0 +1,50 @@
+{
+  "parameters": {
+    "api-version": "2018-11-01-preview",
+    "billingAccountName": "{billingAccountName}",
+    "parameters": {
+      "address": {
+        "companyName": "Contoso",
+        "firstName": "Test",
+        "lastName": "User",
+        "addressLine1": "1 Microsoft Way",
+        "city": "Redmond",
+        "postalCode": "98052",
+        "region": "WA",
+        "country": "US"
+      }
+    }
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Location": "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/patchOperations/65e1f2bf-c31c-4b89-a599-25d9d4172af9?api-version=2018-11-01-preview",
```

Comment:
You shouldn't include location with 202. For Patch and re-Put, to get the status of the change they just Get the resource and provisioningState should be set to the status. If you want a richer status API, use Azure-AsyncOperation.

https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creating-or-updating-resources

Reply:
updated to use Azure-AsyncOperation

Reply:
updated.